src/common/snippets/src/lowered/pass/set_buffer_reg_group.cpp

```diff
@@ -49,18 +50,23 @@ bool SetBufferRegGroup::are_adjacent(const BufferMap::value_type& lhs, const Buf
         return !can_be_in_one_reg_group(lhs.second, rhs.second);
     } else {  // Buffers are connected to the same Loop, but one of Buffers - inside this Loop, another - outside
         // Buffers are adjacent if outer Buffer has non-zero data shift params
-        if (lhs_ids.size() == rhs_ids.size()) // If the count of outer Loops are equal, it means that outer loops are already different
+        if (lhs_ids.size() ==
+            rhs_ids.size())  // If the count of outer Loops are equal, it means that outer loops are already different
```

Comment:
Minor: probably it would be better if comment will be above condition - then condition will be only one line without splitting into two lines

Reply:
Done